### DISCUSSION TITLE
sir, how can we know that on server side particular socket id is successfully emitted or not

### DISCUSSION THREAD

**Discussion by PUNEET-8** [miguelgrinberg/python-socketio#729] [Q&A]
......
.......
receiver_Id, user = await get_receiver_sid(room)

# How to know below emit is successfully emitted 
status = await sio.emit('my_response', {'data': messages}, room=receiver_Id)
# This Status is always return None , whether successfully emmited or not

Actually I want to make the chat where i want to know the receiver message seen or not,

**Comment by miguelgrinberg** [comment]
Did you see how to implement callbacks? It's in the documentation.
